### ISSUE TITLE
Create an action for sending email

### ISSUE THREAD

**Issue by cofiem** [QutBioacoustics/baw-workers#24]
This is take the impact of sending email off the web server.

The action should be very simple.

**Comment by atruskie** [comment]
I disagree with the need for this. We don't really send mass email. Only cases where this actually causes a concern is when our site is generating mass errors... in which case added complexity could destabilize the ability to even notify us of errors.

Good future option if we ever start mailing users with updates.

**Comment by atruskie** [comment]
Related to: https://github.com/QutBioacoustics/baw-private/issues/17

**Comment by atruskie** [comment]
Related to #2 and #32 - I retract my previous statement.
